failures-summary-and-bottle-result/action.yml

```diff
@@ -4,7 +4,7 @@ inputs:
   workdir:
     description: Working directory of the result
     required: true
-    default: ${{ github.workspace }}
+    default: ${{ env.BOTTLES_DIR }}
```

Comment:
This doesn't seem to work: https://github.com/Homebrew/homebrew-core/actions/runs/4795639651/jobs/8531058547

Reply:
Reverted in ac970b34a979d5836e0620ee8c4fce487f0aa2ae.